test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -60,6 +61,30 @@ struct testNumaNodes : public testing::TestWithParam<int> {
         return numa_node;
     }
 
+    long unsigned int retrieve_nodemask(void *addr) {
+        struct bitmask *retrieved_nodemask = numa_allocate_nodemask();
+        EXPECT_NE(nodemask, nullptr);
```

Comment:
Use `UT_ASSERT...` macros for assertions.

Reply:
Done.